lib/Importer/ModelLoader.cpp

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Importer/ModelLoader.h"
+
+#include <cassert>
+#include <string>
+
+using namespace glow;
+
+namespace glow {
+
+unsigned NCHW2NHWC[4] = {0u, 2u, 3u, 1u};
+unsigned NHWC2NCHW[4] = {0u, 3u, 1u, 2u};
+
+bool arrayIsConstant(const llvm::ArrayRef<size_t> a) {
+  for (size_t i = 1; i < a.size(); i++)
+    if (a[0] != a[i])
+      return false;
+  return true;
+}
+
+}; // namespace glow
+
+Tensor *ModelLoader::getTensorByName(const std::string &name) {
+  assert(tensors_.count(name) &&
+         "There is no tensor registered with this name.");
+  return tensors_[name];
+}
+
+Node *ModelLoader::getNodeByName(const std::string &name) {
+  auto it = nodeByName_.find(name);
+  if (it != nodeByName_.end()) {
+    return it->second;
+  }
+
+  llvm_unreachable("Could not find a node with this name.");
```

Comment:
This is bad.  The `llvm_unreachable` can be entirely elided.  Add a `return nullptr` at the end please.

Reply:
Done.